### PR TITLE
propose meeting facilitator role

### PR DESCRIPTION
**Pull request by ultrasaurus**
Adding role for meeting facilitator
starting with co-chairs as meeting facilitators

(also fixed capitalization consistency)

### CODE REVIEW

File: governance/roles.md [hunk 1]
```diff
@@ -145,8 +145,29 @@ and led by a Security Assessment Facilitator, who will:
...
 * triage issues related to security assessments.
 
-
-### Triage Team
+### Meeting facilitator
+
+The group meetings are an important part of community building and the 
+facilitator ensures a welcoming and inclusive atmosphere. In keeping with these
+goals, the meeting facilitator has the following responsibilities:
+* prepares the meeting notes with template and agenda
```
Comment (MVrachev) [inline governance/roles.md]: Considering that a non-chair member can have a `Meeting facilitator` role (as described below) there is duplication with the chair role: 
image

Maybe it makes sense to remove that responsibility from the Chair members and give it to the Meeting facilitators?

File: governance/roles.md [hunk 2]
```diff
@@ -145,8 +145,29 @@ and led by a Security Assessment Facilitator, who will:
...
+* coordinates with presenter, if applicable
+* ensures there are scribes
+* runs meeting check-in, including partner groups
+* leads the meeting through the agenda
+
+Prerequisites:
+* active member
+* history of regular attendance
+* participation as scribe
```
Comment (lumjjb) [inline governance/roles.md]: I'm a little unsure if I parse this correctly... does this mean "Has participated as a scribe before" or "Has to participate as a scribe during the meeting that one is facilitating"?

File: .github/settings.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ collaborators:
 
   - username: pragashj
     permission: admin
-  
+
```
Comment (lumjjb) [inline .github/settings.yml]: nit: spacing

File: governance/roles.md [hunk 4]
```diff
@@ -145,8 +148,32 @@ and led by a Security Assessment Facilitator, who will:
...
+* prior participation as scribe
+
+One or more of the following:
+* served in another facilitator role
+* leader on a project team
+* tech lead
+* co-chair
+
+Any member who meets the above qualifications is encouraged to add themselves
+to the [list of people in this role](.github/settings.yml).
```
Comment (lumjjb) [inline governance/roles.md]: This link is actually going into relative `governance/` probably `/.github/settings.yml` would work?

**Comment by ultrasaurus** [comment]
addressed feedback

**Pull request merged**